src/users/users.ts

```diff
@@ -163,9 +162,9 @@ export class Users {
 
   async sendMagicAuthCode(
     options: SendMagicAuthCodeOptions,
-  ): Promise<MagicAuthChallenge> {
+  ): Promise<{ user: User }> {
     const { data } = await this.workos.post<
-      MagicAuthChallenge,
+      { user: User },
```

Comment:
We'll need to use the serialized representation here and then deserialize appropriately.